assets/dev/scss/frontend/widgets/image-box.scss

```diff
@@ -3,44 +3,20 @@
 //
 .elementor-widget-image-box {
 
-	.elementor-image-box-content {
-		width: 100%; // Hack for Chrome in flex mode
-	}
-
-	@media (min-width: $screen-mobile-next) {
-
```

Comment:
I have applied the image position to all breakpoints, but kept the default value for the mobile version.